client/src/pages/learn/2022/responsive-web-design/learn-intermediate-css-by-building-a-cat-painting/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Introduction to the Learn Intermediate CSS by Building a Cat Painting
+block: learn-intermediate-css-by-building-a-cat-painting
+superBlock: 2022/responsive-web-design
+isBeta: true
```

Comment:
This shouldn't be in beta since the responsive web design curriculum is already live